src/server.js

```diff
@@ -1,21 +1,64 @@
-const express = require('express');
+const express = require("express");
+require("express-async-errors");
+const { Prisma } = require("@prisma/client");
+
 const app = express();
 
-const cors = require('cors');
-const morgan = require('morgan');
+const cors = require("cors");
+const morgan = require("morgan");
 
-app.disable('x-powered-by');
+app.disable("x-powered-by");
 
 // Add middleware
 app.use(cors());
-app.use(morgan('dev'));
+app.use(morgan("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-
 // Tell express to use your routers here
-const customerRouter = require('./routers/customer');
-app.use('/customers', customerRouter);
+const customerRouter = require("./routers/customer");
+const moviesRouter = require("./routers/movies");
+const screensRouter = require("./routers/screens");
+const ticketsRouter = require("./routers/tickets");
+const reviewsRouter = require("./routers/reviews");
+
+app.use("/customers", customerRouter);
+app.use("/movies", moviesRouter);
+app.use("/screens", screensRouter);
+app.use("/tickets", ticketsRouter);
+app.use("/reviews", reviewsRouter);
 
+app.use((e, req, res, next) => {
+  if (e instanceof Prisma.PrismaClientKnownRequestError) {
+    if (e.code === "P2002") {
+      return res
+        .status(409)
+        .json({ error: "A customer with the provided email already exists" });
+    }
+  }
+  if (e instanceof Prisma.PrismaClientKnownRequestError) {
+    if (e.code === "P2013") {
+      return res
+        .status(400)
+        .json({ error: "Missing fields in request body" });
+    }
+  }
+  if (e instanceof Prisma.PrismaClientKnownRequestError) {
+    if (e.code === "P2001") {
+      return res
+        .status(404)
+        .json({ error: "Customer with that id does not exist" });
+    }
+  }
```

Comment:
All of these if statements are the same, you could condense the code checking into one parent `if`:

```js
  if (e instanceof Prisma.PrismaClientKnownRequestError) {
    if (e.code === "P2002") {
      return res
        .status(409)
        .json({ error: "A customer with the provided email already exists" });
    }

    if (e.code === "P2013") {
      return res
        .status(400)
        .json({ error: "Missing fields in request body" });
    }

    if (e.code === "P2001") {
      return res
        .status(404)
        .json({ error: "Customer with that id does not exist" });
    }
  }
```

You could also use a switch statement for the error code here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

```js
if (e instanceof Prisma.PrismaClientKnownRequestError) {
  switch (e.code) {
    case "P2002":
      return res.status(409).json({ error: "A customer with the provided email already exists" })
      break
    
    case "P2013"
      ...
  }
}
```